### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by zRzRzRzRzRzRzR** [THUDM/CogVideo#164] [Ideas]
Hi everyone! 👋  

This **Discussions** tab is the perfect place to collaborate, ask questions, and share ideas **outside of bug reports**! 🚀  

Here, you can:  
* Discuss why the model behaves in certain cases (e.g., performance variations or limitations)  
* Ask questions about fine-tuning the model using [SAT](https://github.com/THUDM/CogVideo/tree/main/sat)
* Brainstorm and ask for help with implementation, workflows, or best practices  

**Please note**: For clear bug reports or feature requests, continue to use the **Issues** section 🐞. Let’s keep Discussions as a hub for learning, exploring, and growing together! 🌱  

Feel free to start a new thread or join existing ones. Happy discussing! 🎉

**Comment by tiandiyihao1997** [comment]
How to fix the noise so that the loss of each fine-tuning is the same?

**Comment by gloryafeks77-art** [comment]
Make it same by same, so that I can use this to generate a video. Thank you.🌙 Nia and the Sleepy Star Forest
(A Calming Bedtime Story for Kids)

Tonight, Nia couldn’t sleep.

She lay in her cozy bed, staring out the window at the quiet night sky.

The stars twinkled softly… but something felt different.

One little star flickered… then dimmed.

Nia sat up.

“That’s strange…” she whispered.

Suddenly, a gentle breeze filled her room, carrying a soft, glowing light.

Before she knew it, Nia felt herself drifting… floating… as if the night itself was guiding her.

And when she opened her eyes…

She was no longer in her room.

She was standing at the edge of a magical forest.

The trees shimmered softly, their leaves glowing like tiny stars.

Above her, the sky sparkled — but many stars looked sleepy… and dim.

A small, glowing creature floated toward her.

It looked like a tiny fox made of starlight.

“Hello, Nia,” it said gently.

“I’m Lumo… the guardian of the Star Forest.”

Nia blinked in wonder.

“The stars… they look tired,” she said softly.

Lumo nodded.

“They are. The Star Forest is where stars come to rest… but lately, they cannot fall asleep.”

Nia looked around.

The forest felt quiet… but uneasy.

“Why can’t they sleep?” she asked.

Lumo lowered his glowing head.

“They are worried… and a little afraid of the dark.”

Nia’s eyes widened.

“The stars… are afraid of the dark?” she whispered.

Lumo gave a small nod.

“They shine so brightly for others… but sometimes, even they forget how to feel safe.”

Nia thought for a moment.

Then she smiled gently.

“I think I can help,” she said.

Lumo’s eyes sparkled.

“Follow me,” he said softly.

They walked deeper into the forest.

All around them, tiny stars flickered nervously, hiding behind glowing leaves.

Nia knelt beside one small star.

“Hello,” she whispered kindly.

The little star trembled.

“It’s too dark… I don’t like it…” it said.

Nia spoke in a calm, gentle voice.

“The dark isn’t something to be afraid of,” she said.

“It’s what helps us rest… and dream.”

The star flickered uncertainly.

“But what if I stop shining?” it asked.

Nia smiled.

“Then you can shine again tomorrow,” she said softly.

“Even the brightest lights need sleep.”

The little star grew a little brighter.

“Really?” it asked.

Nia nodded.

“Of course. You don’t have to shine all the time.”

One by one, other stars began to listen.

They gathered around Nia, their glow soft and curious.

Nia closed her eyes and took a slow, deep breath.

“Let’s try something together,” she said gently.

“Breathe in… slowly…”

The stars shimmered softly.

“And breathe out…”

The forest grew calmer.

Again…

“In… and out…”

One by one, the stars began to relax.

Their light softened… becoming warm and peaceful.

The forest glowed with a gentle, sleepy glow.

Lumo smiled.

“They’re resting,” he whispered.

Nia looked around happily.

The stars were no longer flickering with fear.

They were calm… safe… and ready to sleep.

“You helped them remember,” Lumo said softly.

“Remember what?” Nia asked.

“That it’s okay to rest.”

A soft light began to surround Nia again.

The forest faded gently.

“You’ll always be welcome here,” Lumo whispered.

“And so will anyone who needs a little calm.”

Nia smiled.

“Goodnight,” she said softly.

And as the light carried her away…

She opened her eyes…

Back in her cozy bed.

The night sky outside her window shimmered peacefully.

The stars were glowing softly… resting.

Nia snuggled into her blanket.

She took a slow, gentle breath…

And closed her eyes.

Feeling calm… safe… and ready to dream.

🌙

And somewhere far away…

In the Sleepy Star Forest…

The stars slept peacefully… shining again tomorrow.

Goodnight.

**Comment by gloryafeks77-art** [comment]
MAKE VIDEO

**Comment by muwaddathussain1234-oss** [comment]
Style: "Modern Phonk" or "UK Drill Instrumental."
Search for: “Aggressive Cinematic Hip Hop Beat” or “Dark Car Commercial Beat.”
Sound Effect Tip: Add a subtle "Turbo Spool" or "Deep Engine Roar" sound effect at the 0:02 mark for a professional touch.
![Uploading I_want_same_video_for_202605081718.jpeg…]()
Please make video
